### PR TITLE
HPC-Runner-Command 3.2.1

### PR DESCRIPTION
**Pull request by jerowe**
* [ ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/perl-hpc-runner-command/3.2.1/build.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# If it has Build.PL use that, otherwise use Makefile.PL
+git config --global user.email "you@example.com"
```
Comment (bgruening) [inline recipes/perl-hpc-runner-command/3.2.1/build.sh]: @jerowe why is this needed? Can we add a comment?

File: recipes/perl-hpc-runner-command/3.2.1/build.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+git config --global user.name "Your Name"
+
+cpanm --installdeps .
```
Comment (bgruening) [inline recipes/perl-hpc-runner-command/3.2.1/build.sh]: Is this also needed? It is better to add all missing deps explicitly.

Comment (jerowe) [inline recipes/perl-hpc-runner-command/3.2.1/build.sh]: I'm still working on low level cpanm deps. I have another PR in from yesterday that gets some, and then I have quite a few more ready!

**Comment by jerowe** [comment]
I'm going to close this one out. I have a few bug fixes for 3.2.2 that I should get out by the end of the day anyways.
